patches/colorjs.io+0.4.5.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/node_modules/colorjs.io/dist/color.cjs b/node_modules/colorjs.io/dist/color.cjs
+index 3835d9a..5b8e8e1 100644
+--- a/node_modules/colorjs.io/dist/color.cjs
++++ b/node_modules/colorjs.io/dist/color.cjs
+@@ -2010,14 +2010,14 @@ var P3 = new RGBColorSpace({
+ // Default space for CSS output. Code in Color.js makes this wider if there's a DOM available
+ defaults.display_space = sRGB;
+ 
+-if (typeof CSS !== "undefined" && CSS.supports) {
++if (typeof CSS !== "undefined" && CSS?.supports) {
+ 	// Find widest supported color space for CSS
+ 	for (let space of [lab, REC2020, P3]) {
+ 		let coords = space.getMinCoords();
+ 		let color = {space, coords, alpha: 1};
+ 		let str = serialize(color);
+ 
+-		if (CSS.supports("color", str)) {
++		if (CSS?.supports("color", str)) {
+ 			defaults.display_space = space;
+ 			break;
+ 		}
+@@ -2039,7 +2039,7 @@ if (typeof CSS !== "undefined" && CSS.supports) {
+ function display (color, {space = defaults.display_space, ...options} = {}) {
+ 	let ret = serialize(color, options);
+ 
+-	if (typeof CSS === "undefined" || CSS.supports("color", ret) || !defaults.display_space) {
++	if (typeof CSS === "undefined" || CSS?.supports("color", ret) || !defaults.display_space) {
+ 		ret = new String(ret);
+ 		ret.color = color;
+ 	}
+diff --git a/node_modules/colorjs.io/dist/color.js b/node_modules/colorjs.io/dist/color.js
```

Comment:
We should only need to patch one of these files (either `color.js` or `color.cjs`)

Reply:
`colorio.js`'s `package.json` has the following `exports` defined:

```js
	"exports": {
		".": {
			"types": "./types/index.d.ts",
			"import": "./dist/color.js",
			"require": "./dist/color.cjs"
```

and also in the package json

```js
	"main": "./dist/color.cjs",
	"module": "./dist/color.js",
```

so it seemed safer to do both, but we can pick one if we're certain only that one will be used

Reply:
Our build process will only use one of them to build, so in favor of making the patch as small possible I  think we should patch the one that our build grabs and leave the other alone.

Reply:
Sounds good, switched to only `color.js` then

I'll need to repatch it when https://github.com/dequelabs/axe-core/pull/4449 is merged

Reply:
Nevermind, I rebased my branch so now it'll flow through naturally in theory. Switched patch to be for colorjs.io `0.4.3` instead.